training_track/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Training Tracks
+===============
+
+This module combines the power of modules ``training`` and
+``website_event_track``.
+
+Event tracks can get a training duration type, so it lets the user know if
```

Comment:
s/can get/are assigned
s/so it lets the user know if/informing the user whether